app/views/records/_form.html.erb

```diff
@@ -17,6 +17,7 @@
                  :as => :key,
                  :locals => {:f => f}
       %>
+      <%= f.input :reviewed, :as => :boolean %>
```

Comment:
Hmm, and the ticket doesn't say anything about new assets.  I think this checkbox shouldn't be shown on new assets, just when editing.